addons/sourcemod/translations/mapchooser.phrases.txt

```diff
@@ -0,0 +1,57 @@
+"Phrases"
```

Comment:
don't add the `mapchooser.phrases.txt` file.
put the reroll translations in some other random shavit-x.phrases.txt file like shavit-common, shavit-core, or even shavit-misc.